test/jest.config.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
     '^Test_Runner$': '<rootDir>/../src/Test_Runner.js',
     '^EventInvocation$': '<rootDir>/../src//EventInvocation.js',
     '^IntentReader$': '../src/IntentReader.js',
+    '^CensorData$': '../plugins/censorData.json',
```

Comment:
this might not be correct. we would have to keep the path of json file present in fca